front/poke/temporal/activities.ts

```diff
@@ -42,6 +41,7 @@ import {
 } from "@app/lib/models/assistant/conversation";
 import { ContentFragmentResource } from "@app/lib/resources/content_fragment_resource";
 import { frontSequelize } from "@app/lib/resources/storage";
+import { MembershipModel } from "@app/lib/resources/storage/models/membership";
```

Comment:
this file is the one exception where I kept a raw `MembershipModel` in the business logic. it's very poke specific since we are deleting all data for a workspace.

Reply:
I'd be in favor of using Resource here as well. As per comment below where we break the logic by not using it AFAICT

Reply:
Done